Gopkg.toml

```diff
@@ -31,8 +31,8 @@
   version = "0.8.0"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/prometheus/common"
```

Comment:
how come we changed this? ... most of them seem to have the branch on top

Reply:
now I found it is yolo order anyway

Reply:
all things should have alphabetical order now